### ISSUE TITLE
Cleaning up warnings view

### ISSUE THREAD

**Issue by vlidholt** [spritebuilder/SpriteBuilder#318]
Never happened for the 1.0 release, but should be fixed. The warnings view doesn't look too great. Fixes that should be done:
- There should be no focus ring when the table view is selected (use the CCBTableView in Interface Builder).
- The font should be smaller. Use same font as in the timeline.
- Selecting items should not use standard blue color, but use same selections visuals as in the file list.
- Would be nice to use the yellow warnings triangle in front of each error message.

![screen shot 2014-03-12 at 8 04 10 pm](https://f.cloud.github.com/assets/1539812/2405875/40ae3aae-aa5c-11e3-805c-2a3f02f78f20.png)

![warning](https://f.cloud.github.com/assets/1539812/2405893/cb73a7dc-aa5c-11e3-8739-bce89351d2c6.png)

**Comment by NickyWeber** [comment]
An addition: What about going XCode and this status to the middle of the tool bar, which is quite empty:

![bildschirmfoto 2014-03-13 um 17 17 48](https://f.cloud.github.com/assets/5259653/2411761/2e060b38-aacb-11e3-9c6b-0d637d5b840e.png)

In case of warnings this would show up prominently at the top. A click would open the warnings pane.

**Comment by NickyWeber** [comment]
Added changes in "Feature/#318 cleaning warnings" #337

**Comment by vlidholt** [comment]
Can still use some tweaks (warning icons), but good enough for now. Moving it up a milestone.
